Update README.md (#132)
buildingamind/NewbornEmbodiedTuringTest: README.md

### Installation

1. **Virtual Environment Setup**

>    *(Note: if you are a member of our lab using one of our lab servers, you will need to  initialize Anaconda for your account the first time you use the server for this pipeline: `module load anaconda`)*

Reviewer: Phrasing should be less lab specific
```suggestion
   *(Note: If anaconda is not already initialized, you may need to do so using: `module load anaconda`)*
```

Author: Let's leave this lab specific. We need to leave this information in for our own internal purposes, but we don't know that this will work for other people (it depends on what system they're using and how their servers are configured). Instead of making it sound like this is a general direction for everyone, it's better to make it clear that it's for our internal purposes.

### Running a NETT

1. **Download the Unity Executable**

   You can download and unzip pre-made Unity executables from [here](https://origins-of-intelligence.com/environments/). An executable is required to run the virtual experiments.

   If you are running this pipeline on a server and you need to copy the executable folder from your local machine to the server, use the following:

   ```bash
   scp -r \local\path\to\executable\folder  \server\parth\to\directory\for\executable\folder\
   ```

>    *(Note: if you are a member of our lab using one of our lab servers, the server path will be: your_username@server_name.luddy.indiana.edu:/home/your_username)*

Reviewer: Should we be putting our personal server information on a public repo? I think it might make more sense to hold this in internal documentation (either on our Google Drive or in the buildingamind/docs repo)

Author: Hmm... let's talk about this together at the Wednesday meeting. It's not clear to me that this is sensitive information. The only information that's getting provided is the luddy.indiana.edu part and the fact that there's a home directory, but both of those seem pretty obvious. I really like the idea of having one place to go to for running the pipeline instead of a bunch of different pieces of documentation. But if I'm missing how this is sensitive, then we can move it. Perhaps we can get some opinions from the group on how sensitive this info is?

2. **Import NETT Components**

   Open python.
   ```bash
   python

>    ```

Reviewer: If we are showing them how to run the code in python, we should probably let them know that they can run it in a file, which would be the preferred/easier method.
```suggestion
   From here, everything will be done in Python. Either open the interpreter in interactive mode by running `python` or write the code into a python file, which can be run with `python <FILE>.py`
```

Author: This change works for me!

   ```python
   brain = Brain(policy="CnnPolicy", algorithm="PPO")
   ```

>    *(To get a list of all available policies, algorithms, and encoders, run `nett.list_policies()`, `nett.list_algorithms()`, and `nett.list_encoders()` respectively. You will need to nett to run these commands: `import nett`.)*

Reviewer: ```suggestion
   *(To get a list of all available policies, algorithms, and encoders, run `nett.list_policies()`, `nett.list_algorithms()`, and `nett.list_encoders()` respectively, which requires you to run`import nett` beforehand.)*
```

Author: Great change!

- **Environment**

>    Create the simulation environment using the path to your Unity executable (see Step 1). The path must be absolute and not relative.

Reviewer: That is incorrect. The path can be relative.

Author: That's even easier then! Please feel free to change that.

   ```python
   environment = Environment(executable_path="path/to/executable.x86_64")
   ```

>    *(To get a list of all available configurations, run `nett.list_configs()`. If you have not already imported nett, you will need it to run this command: `import nett`.)*

Reviewer: Line can be deleted since that function no longer exists.
```suggestion
```

Author: Great catch!
